frontend/src/components/elements/DeckGlChart/DeckGlChart.tsx

```diff
@@ -476,7 +475,4 @@ function parseGetters(type: any, spec: any): void {
   })
 }
 
-export default compose(
-  withMapboxToken,
-  withFullScreenWrapper
-)(DeckGlChart)
+export default withMapboxToken(withFullScreenWrapper(DeckGlChart))
```

Comment:
This is much more readable than using LoDash's `flowRight`. Is there are reason to use it here instead?

Reply:
From my point of view is more readable using a compose function as it is very common in the react world, actually redux expose their own compose function and it is also mentioned in the[ react documentation ](https://en.reactjs.org/docs/higher-order-components.html#convention-maximizing-composability)

Reply:
I am a bit of an anti-JavaScript partisan, so take this with a grain of salt, but: I _strongly_ disagree that the compose function is more readable. What's the benefit of a separate function that just composes already-composable functions (and essentially obfuscates that process - I need to go into `compose()` to see what it's actually doing)?

It feels to me like it adds cognitive overhead; adds debugging overhead (I have to step into the compose function if I'm stepping through this code; and stack traces are going to be less meaningful); it _may_ add runtime overhead, probably not but I have to go read the code to find out; and its functionality isn't worth those overheads.

Unless there's a huge code savings, I will almost always come down on the side of explicit being much more desirable than implicit.

(Apologies for the rant; and of course I hope it goes without saying that I definitely think reasonable people can disagree here! This is also quite a small thing; it's more the underlying philosophy it belies that rubs me the wrong way :))

Reply:
Yeah, I'm with Tim on this one (minus the rant! 🤣 )

`foo(bar(baz))` is 100% clear to anyone reading it, while `compose(bar, foo)(baz)` is not. In fact, I don't even know if it should be `compose(foo, bar)(baz)` — and that actually proves that point that the other notation is confusing!

I'll go ahead and push this change for now.